unpackaged/config/kav/package.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Package xmlns="http://soap.sforce.com/2006/04/metadata">
-  <types>
-    <name>Layout</name>
-    <members>Knowledge__kav-FAQ - Author</members>
-    <members>Knowledge__kav-FAQ - Reader</members>
-    <members>Knowledge__kav-Procedure - Author</members>
-    <members>Knowledge__kav-Procedure - Reader</members>
-  </types>
-  <types>
-    <name>CompactLayout</name>
-    <members>Knowledge__kav.Recommended_Compact_Layout</members>
-  </types>
-  <types>
-    <name>CustomTab</name>
-    <members>Knowledge__kav</members>
-  </types>
-  <types>
-    <name>CustomObject</name>
-    <members>Knowledge__kav</members>
-  </types>
-  <version>48.0</version>
-</Package>
\ No newline at end of file
+    <types>
```

Comment:
hmmm not sure why all these package XML changes are in here.... we probably need to be merging branches down manually

Reply:
Yeah was confused too

Reply:
@Julian88Tex @rjanjanam  i did not check in these files